examples/transponder.py

```diff
@@ -10,7 +10,7 @@ async def run():
     await drone.connect()
 
     # Start the tasks
-    tasks = {asyncio.create_task(print_transponders(drone))}
+    _tasks = {asyncio.create_task(print_transponders(drone))}
```

Comment:
It is unusual to assemble these tasks but not await them.  Are we certain that they will be run at some point?